examples/llama2_driver.py

```diff
@@ -93,10 +105,25 @@ def _main():
         top_k=args.top_k,
     )
 
-    if (args.pipeline_parallel and rank == 0) or (not args.pipeline_parallel):
+    if (args.pp and rank == 0) or (not args.pp):
         print(tokenizer.decode(generated_tokens.tolist()))
 
     return 0
 
 if __name__ == '__main__':
     sys.exit(_main())
+
+# To create checkpoints from transformer model:
+#   Use gpt-fast
+#   ./scripts/prepare.sh meta-llama/Llama-2-7b-chat-hf
```

Comment:
I am not sure, why do we need to create checkpoint? One would get llama2 checkpoint from meta.

Reply:
This step is also converting the weights from hf format to torch.nn.module format.

Reply:
Could we eliminate gpt-fast dependency?

Reply:
It's an independent script so we can surely pull that script into orchard. The original source of that script is not even gptfast. It comes from another open source project to convert between different weight formats.

Reply:
Let's do pull the script into orchard.

Reply:
Done!